### PR TITLE
feat: intermediate screen to make sure the private key copying

### PR DESCRIPTION
**Pull request by mastereng12**
## Context

Closes: #43 

- add more commands
- cleanup the repo and advance UX

---

## Reviewers

- @cool-develope 
- @kashishshah

### CODE REVIEW

**Comment by kashishshah** [comment]
@mastereng12 @cool-develope I think we should also update the readme with new information in this same PR.

File: README.md [hunk 1]
```diff
@@ -216,10 +239,6 @@ cd $HOME/.lagrange && docker compose -f <docker-compose-file> down --remove-orph
 
 ```
 
-> NOTE: If you are seeing this error message `time= level=error msg=failed to join the network: failed to join the network: pc error: code = Unknown desc = the operator is not a committee member`, please check if you are using a correct BLS key pair that you created in the `register` step because the node won't be able to join the committee with incorrect BLS keys. If you are using correct BLS keys then it means that the given attestation node setup is successful and it will start attesting from the next epoch. The current epoch period of the state committee rotation is 24 hours for Holesky testnet.
```
Comment (kashishshah) [inline README.md:219]: Did we change the error message when the operator is waiting to join? If not we will need some a note in post deployment so operators don't have to reach out to us

**Comment by kashishshah** [comment]
Overall lgtm!

**Review by kashishshah — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mastereng12) [reply README.md:219]: the error message is updated

**Pull request merged**